session4/src/com/itschool/homework/HomeRun.java

```diff
@@ -5,6 +5,7 @@
 public class HomeRun {
     public static void main(String[] args) {
         //challenge1();
+        //rectangleArea();
```

Comment:
why commented? See also my previous comment here from the previous review done a few days ago.